PatchesAndAreasApi.Tests/Dockerfile

```diff
@@ -13,27 +13,28 @@ ENV SONAR_TOKEN=$SONAR_TOKEN
 WORKDIR /app
 
 # Install and run sonar cloud scanner
-RUN apt-get update && apt-get install -y openjdk-11-jdk
-RUN dotnet tool install --global dotnet-sonarscanner --version 5.6.0
+RUN apt-get update && apt-get install -y openjdk-17-jdk && apt-get clean \
+&& dotnet tool install --global dotnet-sonarscanner
 ENV PATH="$PATH:/root/.dotnet/tools"
 
-RUN dotnet sonarscanner begin /k:"LBHackney-IT_patches-and-areas-api" /o:"lbhackney-it" /d:sonar.host.url=https://sonarcloud.io /d:sonar.login="${SONAR_TOKEN}"
+RUN dotnet sonarscanner begin /k:"LBHackney-IT_patches-and-areas-api" \
+/o:"lbhackney-it" /d:sonar.host.url=https://sonarcloud.io /d:sonar.login="${SONAR_TOKEN}"
 
 # Copy csproj and nuget config and restore as distinct layers
 COPY ./PatchesAndAreasApi.sln ./
 COPY ./PatchesAndAreasApi/PatchesAndAreasApi.csproj ./PatchesAndAreasApi/
 COPY ./PatchesAndAreasApi.Tests/PatchesAndAreasApi.Tests.csproj ./PatchesAndAreasApi.Tests/
 COPY /NuGet.Config /root/.nuget/NuGet/NuGet.Config
 
-RUN dotnet restore ./PatchesAndAreasApi/PatchesAndAreasApi.csproj
-RUN dotnet restore ./PatchesAndAreasApi.Tests/PatchesAndAreasApi.Tests.csproj
+RUN dotnet restore ./PatchesAndAreasApi/PatchesAndAreasApi.csproj \
+&& dotnet restore ./PatchesAndAreasApi.Tests/PatchesAndAreasApi.Tests.csproj
```

Comment:
This can be just the `dotnet restore` as it implicitly restores everything under the solution file.
Alternatively, it can be just the restore of the test project, as this restore would trigger the restore of the main project,